MapCore/src/main/java/org/wwarn/mapcore/client/components/customwidgets/facet/FacetSearchableCheckBoxWidget.java

```diff
@@ -57,11 +62,12 @@
     public static final String STYLE_CHECKBOXLIST_ITEM_CHECKED = "searchCheckBoxListItemChecked";
     public static final String STYLE_CHECKBOXLIST_ITEM_DISABLED = "searchCheckBoxListItemDisabled";
     public static final String STYLE_INITIAL_SEARCHBOX_TEXT = "initialSearchBoxText";
-    public static final String STYLE_SELECTION_CLEAR_ANCHOR = "searchableCheckBoxClearAnchor";
```

Comment:
thanks nigel, I was going to remove this comment after reviewing all, and make sense to move the class to GSS.